plugins/zns/zns.h

```diff
@@ -18,6 +18,7 @@ PLUGIN(NAME("zns", "Zoned Namespace Command Set", NVME_VERSION),
 		ENTRY("open-zone", "Open one or more zones", open_zone)
 		ENTRY("offline-zone", "Offline one or more zones", offline_zone)
 		ENTRY("set-zone-desc", "Attach zone descriptor extension data to a zone", set_zone_desc)
+		ENTRY("flush-zone", "Flushes a range of logical blocks from a ZRWA to a zone.", flush_zone)
```

Comment:
The official name in the specification is "Flush Explicit ZRWA". Can this be renamed to "zrwa-flush" (or even zrwa-exp-flush) to better represent the functionality? The naming should then also be propagated to the function names, documentation and so forth.